app/package.json

```diff
@@ -11,7 +10,14 @@
     "build-dev": "webpack --mode development",
     "start": "webpack-dev-server --mode development",
     "start-build-time-profile": "webpack-dev-server --mode development --profile --progress",
-    "prod-server": "npx http-server -p 3000 --proxy http://localhost:3000\\? dist"
+    "prod-server": "npx http-server -p 3000 --proxy http://localhost:3000\\? dist",
+    "dev": "electron-webpack dev",
+    "compile": "electron-webpack",
+    "dist": "yarn compile && electron-builder",
+    "dist:dir": "yarn dist --dir -c.compression=store -c.mac.identity=null"
```

Comment:
we need better names for that. It's not clear now which commands output web and which electron. I'd use `electron:dev` instead of `dev`. We can do the same for web, like `web:start`